### PR TITLE
Feat/gray background

### PR DESCRIPTION
**Pull request by gtasautumn**
This is now a bucket for tons of bug fixes, refactoring, and global style updates.

Make sure you run yarn install.

### CODE REVIEW

**Comment by gtasautumn** [comment]
Converting to draft. Gonna include updates for the Title update with this one since it has more side effects than I thought. Broke more tabs that I realized. Will ping you when its good to PR.

File: src/components/seatChart/seatChartCard/SeatChartCard.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                inputType="label"
+                inputVal={item.value}
+              />
+            </Col>
+          </Row>
+        ))}
+
+      <Row>{props.link}</Row>
+
+      <Link to={`/gtas/flightpax/${props.flightId}`}>Flightpax</Link>
```
Comment (kasimhelil) [inline src/components/seatChart/seatChartCard/SeatChartCard.js:28]: I think the intention here is to display the link passed as a prop, like line 26; right? 
Currently, link is not passed for both flight and passenger info

Comment (gtasautumn) [reply src/components/seatChart/seatChartCard/SeatChartCard.js:28]: bug! Should be passed in and the card should position it at the bottom. Can you take care of that for me? I am knee deep right now. If not, I will get to it tonight.

Comment (kasimhelil) [reply src/components/seatChart/seatChartCard/SeatChartCard.js:28]: > bug! Should be passed in and the card should position it at the bottom. Can you take care of that for me? I am knee deep right now. If not, I will get to it tonight.

I got it!

**Review by kasimhelil — APPROVED** [review]
Looks good

**Comment by kasimhelil** [comment]
It should be good to be merged; I have tested all the basic functionalities

**Pull request merged**